packages/js/components/src/select-control/select-control.tsx

```diff
@@ -0,0 +1,162 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { createElement } from 'react';
+import { useCombobox, useMultipleSelection } from 'downshift';
+import { useState, Fragment } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { ChildrenType, ItemType } from './types';
+import { SelectedItems } from './selected-items';
+import { ComboBox } from './combo-box';
+import { Menu } from './menu';
+import { MenuItem } from './menu-item';
+import {
+	itemToString as defaultItemToString,
+	getFilteredItems as defaultGetFilteredItems,
+} from './utils';
+
+type SelectControlProps = {
+	children?: ChildrenType;
+	items: ItemType[];
+	label: string;
+	initialSelectedItems?: ItemType[];
+	itemToString?: ( item: ItemType | null ) => string;
+	getFilteredItems?: (
+		allItems: ItemType[],
+		inputValue: string,
+		selectedItems: ItemType[]
+	) => ItemType[];
+	multiple?: boolean;
+	onInputChange?: ( value: string | undefined ) => void;
+	onRemove?: ( item: ItemType ) => void;
+	onSelect?: ( selected: ItemType ) => void;
+	placeholder?: string;
+	selected: ItemType | ItemType[] | null;
+};
+
+export const SelectControl = ( {
+	children = ( {
+		items,
+		highlightedIndex,
+		getItemProps,
+		getMenuProps,
+		isOpen,
```

Comment:
Can we also expose the `getMenuClassname` and `getItemClassname` maybe through the `getProps` functions for each.
It be nice to expose those instead of having to manually copy the existing css names just to match the styling when a user passes their own children in.

Reply:
Hmm ok, I was not aware that those were available and exported as well.
It would be worth to add an example of using those in the README as well, this is where I was mostly looking (could be in a separate PR of-course)